samples/src/components/MediaDevices.jsx

```diff
@@ -70,13 +70,15 @@ export default (props) => {
         </Col>
         <Col>
           <Button
+            variant={audioStatus.muted?"danger" : 'primary'}
             onClick={() => {
-              props.muteAudioTrack();
+              setAudioStatus({...audioStatus, muted: !audioStatus.muted})
+              props.muteAudioTrack(audioStatus.muted);
             }}
           >
-            Mute
+            {audioStatus.muted?"unMute" : 'Mute'}
```

Comment:
spacing, inconsistent quotes

Reply:
i need to actually add jsx eslint in the samples , have as a todo